### PR TITLE
add text library session in codeigniter to article

### PR DESCRIPTION
**Pull request by coz-git**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/php/sessions/index.md [hunk 1]
```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
 
 #### More Information:
 * <a href="https://secure.php.net/manual/en/book.session.php">php.net session manual</a>
+
```
Comment (co16353sidak) [inline guide/english/php/sessions/index.md]: As a suggestion, firstly, when you are referencing codeignintqr here there is a need to give a basic definition of the same, not all php  beginners will know of the codeigniter framework leading them to do a google search removing their focus away from the article.

Add a small description about codeigniter first.

File: guide/english/php/sessions/index.md [hunk 2]
```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
...
+# library session in codeigniter
+codeigniter have library session who help us for making session
```
Comment (co16353sidak) [inline guide/english/php/sessions/index.md]: this line could be more grammatically correct.
`codeigniter have library session who help us for making session` ->
`codeigniter provides us with more functionality with sessions with it's in-built libraries session functions. `

File: guide/english/php/sessions/index.md [hunk 3]
```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
...
+it have 2 main session like flashdata and temp_data
```
Comment (co16353sidak) [inline guide/english/php/sessions/index.md]: `have` -> `has`

File: guide/english/php/sessions/index.md [hunk 4]
```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
...
+a) flash data is session data that will only be available for the next request, and then automatically cleared.
+b) tempdata is session data with a specific expiration time. After the value expires, or the session expires or is deleted, the value is automatically removed.
+
+To mark an existing item as “tempdata”, simply pass its key and expiry time (in seconds!) to the mark_as_temp() method:
+
```
Comment (co16353sidak) [inline guide/english/php/sessions/index.md]: Also add an example for flashdata alongside the temp_data example.

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@criedchild 
I have reviewed and given some suggestions, please consider them and make the changes.

File: guide/english/php/sessions/index.md (hunk 2 above)
Comment (RandellDawson) [reply guide/english/php/sessions/index.md]: How about a more concise version like:
> codeigniter has a built-in sessions library which makes it easy to work with sessions.`

Comment (co16353sidak) [reply guide/english/php/sessions/index.md]: Yes @RandellDawson and how about in the end we could make it like:- 
`easy to work with sessions` -> `easy to work with sessions and also add more functionality`

Comment (co16353sidak) [reply guide/english/php/sessions/index.md]: as this not only makes default session handling easier but also allows for some additional functionality, so this sounds a little better to me.

Comment (RandellDawson) [reply guide/english/php/sessions/index.md]: Maybe give a few examples in parentheses like:
> easy to work with sessions and also adds more functionality (i.e. list a couple of the big additions here).

**Comment by thecodingaviator** [comment]
@RandellDawson This is stale I guess

**Review by theNaut1lus — COMMENTED** [review]
@RandellDawson, please mark as stale and close (no activity in 21 days.)

**Comment by RandellDawson** [comment]
HI @criedchild 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**